src/commands/hld/append-variable-group.decorator.json

```diff
@@ -0,0 +1,5 @@
+{
+  "command": "append-variable-group <variable-group-name>",
+  "alias": "av",
```

Comment:
Can we change this to `avg` to keep it consistent with the `cvg` we have for project?